src/index.js

```diff
@@ -67,4 +67,14 @@ client.on('messageCreate', (message) => {
     }
 })
 
+client.on('guildMemberAdd', async (member) => {
+    const userInfo = await UserController.getUserByDiscordID(member.id)
+
+    if (userInfo) {
+        member.setNickname(userInfo.firstName + " " + userInfo.lastName);
+    } else {
+        console.log(`Discord user ${newMember.id} not linked, ignoring voice state change.`);
```

Comment:
I think you missed updating the message here

Reply:
thanks for catching